.gitignore

```diff
@@ -1,2 +1,4 @@
 node_modules
 dist
+.DS_Store
```

Comment:
Should be in a user global `.gitignore` file (https://help.github.com/en/articles/ignoring-files#create-a-global-gitignore)

Reply:
This has been removed. Thanks for the explanation :)